### ISSUE TITLE
[AnimeBytes] Default season numbering is incorrect

### ISSUE THREAD

**Issue by DevYukine** [Jackett/Jackett#11860]
### Environment

**OS**:  Docker

**Jackett Version**: v0.18.225

**Last Working Jackett Version**: N/A

**Are you using a proxy or VPN?** [yes/no] no but unrelated

### Description

AnimeBytes doesn't store Season information in there shows but in line https://github.com/Jackett/Jackett/blob/f47c216348fd52f4db2c6b6112334a722a982b9b/src/Jackett.Common/Indexers/AnimeBytes.cs#L241 jackett defaults to S1 for everything which doesn't match any Season/Epsiode RegEx, which is bad because Sonarr doesn't check the specific Series title when it finds a S1 while parsing e.g.
`[-​ZR-​] DATE A LIVE III S01 [Blu-​ray][MKV][h264][1080p][TrueHD 2.​0][Dual Audio][Softsubs (-​ZR-​)` Date A Live III is Season 3 which sonarr normally correctly identifies but because of the default value and the Regex which isn't matching it default to S1 and sonarr would now wrongly map the episodes to S1 even tho its S3

I would really recommend to remove the default, after talking to people in there IRC they recommended to use anidb but i assume this would be out of scope for you guys, if you think removing the default is enough i would be willing to make a small pr 😄  

### Logged Error Messages

None

### Screenshots

Not needed

**Comment by garfield69** [comment]
This is a problematic issue for this web site, because  if you drop the addition of S01 from all torrents, while you may get satisfaction from Sonarr for this one particular series it would likely fail for so many others.
But since none of the team have an account on the site, we cant research on our own and are dependant on volunteers to pick the best approach that provides the maximum matching.
@seeyabye @yuri-moens  you both have had previous dealings with this indexer, any feedback or a PR for this topic?

**Comment by seeyabye** [comment]
This issue is a little more complicated. The S01 is there for a reason and I've experimented with its removal before, but it breaks a lot of cases for Sonarr. Almost all animes that are not seasonal will no longer match on Sonarr's end because Sonarr expects SXX to be there if it's a seasonal pack, or EXX if it's an episode.

Animes with additional seasons are special cases and it's for this reason I proposed https://github.com/Jackett/Jackett/pull/9719 but it will probably not be merged anytime. I say that this is a special case because in some cases, Jackett *can* match the correct season and will return S02, S03 etc. The best case is S01 is replaced with the right season, but it's an impossible task until this protocol is finalized between Sonarr and Jackett. Since there's a conflict in the attribute used, this will unlikely be solved anytime soon.

In conclusion, S01 default is needed to maximize compatibility with Sonarr, and I don't think there's any other way around that. In the given example, if you remove S01, Sonarr will still fail to match because no seasons are matched. An additional option to remove it can be added, but then it will not be useful to Sonarr because of the requirement from Sonarr's end.

If you have a better idea, please let me know.
